app/src/main/res/values-h456dp/integers.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
We can't use size buckets, because both ES1 and ES5 fall into the `large` bucket.  The choice between height and width is largely arbitrary, but for some reason the height differences feel slightly more clear and recognizable to me (`ES5 480h -> ES1 600h -> ES2 800h` vs `ES5 960w -> ES1 1024w -> ES2 1280`).